tests/test_data/sample_evaluation_summary_1.yaml

```diff
@@ -2,6 +2,24 @@ per_template:
   list_all_transformers_within_Substation_SUBSTATION:
     number_of_error_samples: 0
     number_of_success_samples: 10
+    answer_eval_recall:
+      max: 0
+      mean: 0
+      median: 0
+      min: 0
+      sum: 0
```

Comment:
Omit these if no reference answers were supplied?

Reply:
Yes, please. That would be great. Unless there are reference answers I don't see why we would like to serialize zeros here.